doc/security/specs/secure_boot/README.md

```diff
@@ -77,13 +77,31 @@ Devices produced before the key is known to be compromised are not protected by
 
 Additionally, each key is restricted to one of three "roles", which determine in which device states the key can be used.
 The roles are:
-* dev (development, only for devices in the `DEV` lifecycle state)
-* test (manufacturing and testing, only for devices in the `TEST_UNLOCK` lifecycle state)
-* prod (production, only for devices in the `PROD` or `PROD_END` lifecycle states)
+* `dev`: development, only for devices in the `DEV` lifecycle state
+* `test`: manufacturing and testing, only for devices in the `TEST_UNLOCK` lifecycle state
+* `prod`: production, only for devices in the `PROD` or `PROD_END` lifecycle states
+The following table summarizes which role can be used in which lifecycle state:
+
+| Key Type | LC_TEST | LC_RMA | LC_DEV | LC_PROD |
+|----------|---------|--------|--------|---------|
+| `test`   | X       | X      |        |         |
+| `dev`    | X       | X      | X      |         |
+| `prod`   | X       | X      | X      | X       |
 
 If the key indicated in the manifest has a role that doesn't match the lifecycle state of the device, the boot fails.
 All of these keys are 3072-bit RSA public keys with exponent e=65537 (the “F4 exponent”).
 
+The `ROM` has `N` key slots (the exact number depends on the `ROM`) numbered from `0` to `N-1`.
+The `CREATOR_SW_CFG_SIGVERIFY_RSA_KEY_EN` item in the [OTP](otp-mmap) can be used to invalidate a key at
+production time. This item consists of several little-endian 32-bit words. Each word contains four 8-bit hardened booleans
```

Comment:
s/production time/manufacturing time/ to disambiguate. The production term is sometimes used to refer to in-field deployed devices.